api/v1alpha1/healthcheck_types.go

```diff
@@ -159,6 +167,15 @@ type TCPActiveHealthChecker struct {
 	Receive *ActiveHealthCheckPayload `json:"receive,omitempty" yaml:"receive,omitempty"`
 }
 
+// GRPCActiveHealthChecker defines the settings of the GRPC health check.
+type GRPCActiveHealthChecker struct {
+	// ServiceName to send in the health check request.
+	// If this is not specified, then the health check request applies to the entire
+	// server and not to a specific service.
+	// +optional
+	ServiceName *string `json:"serviceName,omitempty" yaml:"serviceName,omitempty"`
```

Comment:
probably merged this too soon, 
thoughts on renaming this to `Service` to keep it consistent with gateway api 
https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io/v1.GRPCMethodMatch

Reply:
I used `ServiceName` to be consistent with the Envoy Proxy API: https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/health_check.proto#envoy-v3-api-msg-config-core-v3-healthcheck-grpchealthcheck 

I can open a new PR to change to `Service` if you want.

Reply:
+1 to being more consistent with GW-API

Reply:
thanks @liorokman +1 to renaming to `Service`